react-app/src/app/screens/Game/components/Square/styles.scss

```diff
@@ -0,0 +1,18 @@
+.square {
+  background: #fff;
```

Comment:
Add a new file in src/scss which contains the colors constants, and you should import those constants in each scss. That way you can use the same colors for the whole app